src/commands/edge/drivers/package.ts

```diff
@@ -63,15 +65,51 @@ export default class PackageCommand extends EdgeCommand {
 		upload: flags.string({
 			char: 'u',
 			description: 'upload zip file previously built with --build flag',
-			exclusive: ['build'],
+			exclusive: ['build-only'],
+		}),
+		assign: flags.boolean({
+			char: 'a',
+			description: 'ask user for a channel to assign the driver to after upload',
+			exclusive: ['assign-to', 'build-only'],
+		}),
+		'assign-to': flags.string({
+			description: 'automatically assign driver to specified channel after upload',
+			exclusive: ['assign', 'build-only'],
+		}),
+		install: flags.boolean({
+			char: 'I',
+			description: 'ask user for hub to install to after assigning it to the channel, implies --assign if --assign or --assign-to not included',
+			exclusive: ['install-to', 'build-only'],
+		}),
+		'install-to': flags.string({
+			description: 'automatically install driver to specified hub, implies --assign if --assign or --assign-to not included',
+			exclusive: ['install', 'build-only'],
 		}),
 	}
 
 	static examples = [
-		'smartthings edge:drivers:package                           # build and upload driver found in current directory',
-		'smartthings edge:drivers:package my-driver                 # build and upload driver found in the my-driver directory',
-		'smartthings edge:drivers:package -b driver.zip my-package  # build the driver in the my-package directory and save it as driver.zip',
-		'smartthings edge:drivers:package -u driver.zip             # upload the previously built driver found in driver.zip',
+`# build and upload driver found in current directory:
+$ smartthings edge:drivers:package
+
+# build and upload driver found in current directory:
+$ smartthings edge:drivers:package
+
+# build and upload driver found in current directory then assign it to a channel (ask user
+# which) and install it to a hub (ask user which)
+$ smartthings edge:drivers:package -I
+
+# build and upload driver found in current directory then assign it to the specified channel
+# and install it to the specified hub
+$ smartthings edge:drivers:package --assign-to <channel-id> --install-to <hubId>
+
+# build and upload driver found in the my-driver directory
+$ smartthings edge:drivers:package my-driver
+
+# build the driver in the my-package directory and save it as driver.zip
+$ smartthings edge:drivers:package -b driver.zip my-package
+
+# upload the previously built driver found in driver.zip
+$ smartthings edge:drivers:package -u driver.zip`
```

Comment:
I'm curious what you guys think of this format...I got it from code in Heroku's CLI. I don't love that it's kinda ugly but I really like that it's easy to edit and see pretty close to what it's gonna look like in use. (In the the output, it gets indented two spaces.)

https://github.com/heroku/cli/blob/e2096b1763d0c93e4ec4779dbe0e3bdd8f0c495b/packages/apps/src/commands/domains/index.ts#L16

Reply:
It also lends itself better to longer descriptions than the two column format does. I think it's good. Lint doesn't seem to like it though ;-)

Reply:
Yeah I think the benefits outweigh the formatting in this case. I like it.

Reply:
I always forget to run the linter. I fixed the linting errors with slight changes to the first and last lines.